.bazelrc

```diff
@@ -24,23 +24,46 @@ build:noninteractive --keep_going
 build:release --config=noninteractive
 build:release --stamp
 
+# Platform-specific options for each supported platform.
+build:remote_linux_x64 --extra_execution_platforms=//platform/linux_x64
+build:remote_linux_x64 --extra_toolchains=//platform/linux_x64:cc-toolchain
+build:remote_linux_x64 --host_platform=//platform/linux_x64
+build:remote_linux_x64 --platforms=//platform/linux_x64
+
+build:remote_macos_x64 --extra_execution_platforms=//platform/macos_x64
+build:remote_macos_x64 --host_action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
+build:remote_macos_x64 --host_platform=//platform/macos_x64
+build:remote_macos_x64 --macos_minimum_os=12
+build:remote_macos_x64 --platforms=//platform/macos_x64
+
+build:remote_windows_x64 --extra_execution_platforms=//platform/windows_x64
+build:remote_windows_x64 --extra_toolchains=//platform/windows_x64:cc-toolchain
+build:remote_windows_x64 --host_platform=//platform/windows_x64
+build:remote_windows_x64 --platforms=//platform/windows_x64
+
+# Options for EngFlow remote execution.
 build:engflow_common --jobs=40
 build:engflow_common --define=EXECUTOR=remote
 build:engflow_common --experimental_inmemory_dotd_files
 build:engflow_common --experimental_inmemory_jdeps_files
 build:engflow_common --incompatible_strict_action_env=true
 build:engflow_common --remote_timeout=600
-build:engflow_common --legacy_important_outputs=false
+build:engflow_common --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
 build:engflow_common --grpc_keepalive_time=30s
 build:engflow_common --experimental_remote_cache_compression=true
-build:engflow_common --remote_instance_name=auth
-build:engflow_common --bes_instance_name=auth
-build:engflow_common --bes_lifecycle_events
+build:engflow_common --remote_download_minimal
+
+build:engflow_bes --bes_backend=grpcs://opal.cluster.engflow.com
+build:engflow_bes --bes_results_url=https://opal.cluster.engflow.com/invocations/auth/
+build:engflow_bes --bes_instance_name=auth
+build:engflow_bes --bes_lifecycle_events
 
-build:opal --config=engflow_common
-build:opal --bes_backend=grpcs://opal.cluster.engflow.com
-build:opal --bes_results_url=https://opal.cluster.engflow.com/invocations/auth/
-# TODO(CUS-349): enable remote execution
+build:engflow --config=engflow_common
+build:engflow --config=engflow_bes
+build:engflow --remote_executor=grpcs://opal.cluster.engflow.com
```

Comment:
Note: glass will be faster, because the CI runner is on glass, so it will be faster if the RE action is on the same cluster. Fine to switch in a followup (or create separate configs for glass and opal so that the switch for both CI Runners and RE can be done more easily).